### PR TITLE
feat(k4): adding pagination

### PR DESCRIPTION
**Pull request by Jeremiapane**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+      <ul class="pagination">
+        <li class="page-item">
+          <a class="page-link" href="#" aria-label="Previous">
+            <span aria-hidden="true">&laquo;</span>
+          </a>
+        </li>
+        <li class="page-item"><a class="page-link" href="#">1</a></li>
+        <li class="page-item"><a class="page-link" href="#">2</a></li>
+        <li class="page-item"><a class="page-link" href="#">3</a></li>
+        <li class="page-item">
```
Comment (Zulqs) [inline index.html:19]: Why this page-item contain nothing?

File: index.html [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+  </head>
+  <body>
+    <nav aria-label="Page navigation example">
+      <ul class="pagination">
+        <li class="page-item">
+          <a class="page-link" href="#" aria-label="Previous">
+            <span aria-hidden="true">&laquo;</span>
+          </a>
+        </li>
+        <li class="page-item"><a class="page-link" href="#">1</a></li>
```
Comment (MEs124) [inline index.html]: tambahin btn

File: index.html (hunk 1 above)
Comment (Jeremiapane) [reply index.html:19]: its not empty. there are a number on the list

File: index.html (hunk 2 above)
Comment (Jeremiapane) [reply index.html]: do you mean change tag a to tag button? done change it

**Review by MEs124 — APPROVED** [review]

**Review by Zulqs — APPROVED** [review]

**Pull request merged**